.changes/unreleased/Features-20221003-110705.yaml

```
@@ -0,0 +1,7 @@
+kind: Features
+body: extend -f
```

Can you expand this to be a bit more specific on what's being added.  A user should be able to read the changelog message and know generally what's been updated.

```suggestion
body: extend -f flag shorthand for seed command
```